doc/nrf/doc_build_process.rst

```diff
@@ -0,0 +1,78 @@
+.. _doc_build_process:
+
+Documentation build process
+###########################
+Whenever the main branch of the `sdk-nrf`_ repository is updated or a pull request is made, a new version of the documentation is built and published. The following section describes the steps involved with building and publishing the documentation.
+
+Building with Sphinx
+********************
+The |NCS| documentation uses the Sphinx framework to build the docsets for HTML.
+Sphinx primarily uses three files to work:
+
+* A makefile that runs the :code:`sphinx-build` command in the correct folder. This folder should contain the next two files.
+* A file :file:`index.rst` that serves as the top of the ToC tree and the root of the documentation.
+* A file :file:`conf.py` in which all further configuration of Sphinx is placed.
+
+The |NCS| documentation does not use a makefile for building Sphinx, and instead uses Ninja. Ninja is a faster version of makefile with less features and complexity, but :file:`.ninja` files are not designed to be hand-written.
```

Comment:
we don't use Ninja, but CMake. Ninja is the recommended CMake generator because it is multiplatform.

Reply:
All references to ninja has been removed